cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -24,6 +24,271 @@ Cask Data Application Platform Release Notes
    :depth: 2
 
 
+`Release 3.0.1 <http://docs.cask.co/cdap/3.0.1/index.html>`__
+=============================================================
+
+
+New Features
+------------
+
+- Added the CDAP Application Templates (``cdap-app-templates``) to the `CDAP Javadocs 
+  <javadocs/index.html>`__
+  (`CDAP-2643 <https://issues.cask.co/browse/CDAP-2643>`__).
+  
+- In the CDAP UI, mandatory parameters for Application Template creation are marked with
+  asterisks, and if a user tries to create a template without one of those parameters, the
+  missing parameter is highlighted
+  (`CDAP-2499 <https://issues.cask.co/browse/CDAP-2499>`__).
+
+
+Improvements
+------------
+
+**Tools**
+
+- Added a tool (`HBaseQueueDebugger 
+  <https://github.com/caskdata/cdap/blob/release/3.0/cdap-master/src/main/java/co/cask/cdap/data/tools/HBaseQueueDebugger.java>`__)
+  that counts consumed and unconsumed entries in an HBase queue
```

Comment:
No, the name means "debugger for queues implemented using HBase tables". We have lots of classes like HBaseQueue* and LevelDBQueue*.